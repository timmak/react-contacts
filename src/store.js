import reducers from "./reducers";
import history from "./history";
import { routerMiddleware } from "react-router-redux";
import { createStore, applyMiddleware } from "redux";
import logger from "redux-logger";
const middleware = routerMiddleware(history);

export default createStore(reducers, applyMiddleware(middleware, logger));
