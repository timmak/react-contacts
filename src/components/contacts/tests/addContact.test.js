import React from "react";
import renderer from "react-test-renderer";
import AddContact from "./../addContact";
import { Provider } from "react-redux";
import store from "./../../../store";

it("should match snapshot", () => {
  const addContact = renderer.create(
    <Provider store={store}>
      <AddContact />
    </Provider>
  );
  const tree = addContact.toJSON();
  expect(tree).toMatchSnapshot();
});
