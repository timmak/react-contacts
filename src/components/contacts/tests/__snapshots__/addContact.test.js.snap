// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <h1>
    Add contact
  </h1>
  <form
    onSubmit={[Function]}
  >
    <div
      className="contact-form--fields"
    >
      <div>
        <label
          className="contact-form--label"
          for="firstName"
        >
          First Name
        </label>
        <input
          className="contact-form--input"
          name="firstName"
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div>
        <label
          className="contact-form--label"
          for="lastName"
        >
          Last Name
        </label>
        <input
          className="contact-form--input"
          name="lastName"
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div>
        <label
          className="contact-form--label"
          for="email"
        >
          Email
        </label>
        <input
          className="contact-form--input"
          name="email"
          onChange={[Function]}
          type="email"
          value=""
        />
      </div>
    </div>
    <div
      className="flex"
    >
      <span
        className="spacer"
      />
      <input
        className="button"
        type="submit"
        value="Save"
      />
    </div>
  </form>
</div>
`;
